kibana.md

```diff
@@ -1101,3 +1101,201 @@ an admin, who has just configured access for a new user. They would like
+| metadata.currentGroup                | string                                                                 | 'administrators'                                                             | Currently selected group by the user                                                 |
+| metadata.availableGroups             | string[]                                                               | [ 'administrators', 'infosec', 'template' ],                                 | List of the all available groups                                                     |
+| metadata.kibanaAccess                | 'ro_strict' or 'ro' or 'rw' or 'admin' or 'unrestricted' or 'api_only' | 'admin'                                                                      | User kibana access                                                                   |
+| metadata.kibanaIndex                 | string or undefined                                                    | '.kibana_administrators'                                                     | Currently selected group kibana index defined in the ACL                             |
+| metadata.kibanaTemplateIndex         | string or undefined                                                    | '.kibana_template'                                                           | Kibana template index defined in the kibana.yml readonlyrest_kbn.kibanaTemplateIndex |
+| metadata.origin                      | string or undefined                                                    | 'Connector name'                                                             | The value is taken from JWT's x-ror-origin claim when ror_kbn_auth is matched        |
+| metadata.impersonatedBy              | string or undefined                                                    | 'admin'                                                                      | Name of the impersonator                                                             |
+| metadata.correlationId               | string                                                                 | 'd7df607d-4b04-454b-979c-3e962e312a7d'                                       | Current session correlationId                                                        |
+| metadata.customMetadata              | Record<string, unknown>                                                | { metadata1: value }                                                         | Custom metadata defined in the ACL                                                   |
+| metadata.allowedApiPaths.http_method | "ANY" or Method                                                        | 'GET'                                                                        | HTTP method defined as allowed in the ACL                                            |
```

Comment:
Do they really have the opportunity to modify `allowedApiPaths` at this stage? Or is it already enforced?

Reply:
`allowedApiPaths` will be called before the custom middleware